sources/dylan/apple-dylan-test-suite/test-range.dylan

```diff
@@ -50,11 +50,11 @@ end test range-4;
 // not except through and up-to in a single expression
 //
 define test range-5 (description: "Through:, by:, and size:")
```

Comment:
This used to test ``through:`` but now tests ``to:``, right?

So it probably makes sense to:

1. Test these error conditions.
2. Make sure that the correct cases are tested as well and return the correct set of values.